src/main/java/frc/robot/subsystems/drive/CommandSwerveDrivetrain.java

```diff
@@ -25,6 +34,12 @@
 public class CommandSwerveDrivetrain extends SwerveDrivetrain implements Subsystem {
     private double vx, vy, omega = 0.0;
     private boolean fieldCentric = true;
+    private boolean aligning = false;
+
+    private Supplier<Pose2d> getFieldToRobot = () -> new Pose2d();
+    private Translation2d fieldToSpeaker = FieldConstants.fieldToRedSpeaker;
+
+    private PIDController thetaController = new PIDController(.5, 0, 0);
```

Comment:
No change required at this time - just wanted to comment that I'm surprised that all you need is a P controller. I imagine IRL we will need integral